cdatgui/editors/secondary/editor/marker.py

```diff
@@ -17,8 +17,8 @@ def __init__(self):
         row = QtGui.QHBoxLayout()
 
         # create type combo box
-        type_box = QtGui.QComboBox()
-        type_box.addItems(["dot", "plus", "star", "circle", "cross", "diamond", "triangle_up", "triangle_down",
+        self.type_box = QtGui.QComboBox()
+        self.type_box.addItems(["dot", "plus", "star", "circle", "cross", "diamond", "triangle_up", "triangle_down",
```

Comment:
We should probably take a page out of the fillarea editor's book and do a visual combobox for these, with an icon for each image. I'll write a script to generate those images for you.